packages/mirage-console-xen-proto/mirage-console-xen-proto.2.3.0/opam

```diff
@@ -0,0 +1,22 @@
+opam-version: "1.2"
+maintainer:    "anil@recoil.org"
+homepage:      "https://github.com/mirage/mirage-console"
+bug-reports:   "https://github.com/mirage/mirage-console/issues"
+dev-repo:      "https://github.com/mirage/mirage-console.git"
+doc:           "https://mirage.github.io/mirage-console/"
+authors:       [ "Anil Madhavapeddy" "David Scott"]
+tags:          [ "org:mirage" "org:xapi-project"]
+license:       "ISC"
+
+build: [
+  [ "jbuilder" "subst"] {pinned}
+  [ "jbuilder" "build" "-p" name "-j" jobs ]
+]
+
+depends: [
+  "ocamlfind" {build}
+  "jbuilder"  {build & >="1.0+beta9"}
+  "mirage-console-lwt" {>= "2.2.0"}
+  "rresult"
+  "xenstore"
+]
```

Comment:
also needs an available ocaml-version >=4.03.0 due to result use:

```
#=== ERROR while installing mirage-console-xen-proto.2.3.0 ====================#
# opam-version         1.2.2 (58ef3b8213100953848d362f7120a30356d7f77d)
# os                   linux
# command              jbuilder build -p mirage-console-xen-proto -j 2
# path                 /home/opam/.opam/4.02.3/build/mirage-console-xen-proto.2.3.0
# compiler             4.02.3
# exit-code            1
# env-file             /home/opam/.opam/4.02.3/build/mirage-console-xen-proto.2.3.0/mirage-console-xen-proto-29-f4ab61.env
# stdout-file          /home/opam/.opam/4.02.3/build/mirage-console-xen-proto.2.3.0/mirage-console-xen-proto-29-f4ab61.out
# stderr-file          /home/opam/.opam/4.02.3/build/mirage-console-xen-proto.2.3.0/mirage-console-xen-proto-29-f4ab61.err
### stderr ###
#     ocamldep xen/proto/mirage_console_xen_proto.dependsi.ocamldep-output
#     ocamldep xen/proto/mirage_console_xen_proto.depends.ocamldep-output
#       ocamlc xen/proto/conproto.{cmi,cmti} (exit 2)
# (cd _build/default && /home/opam/.opam/4.02.3/bin/ocamlc.opt -w -40 -g -bin-annot -I /home/opam/.opam/4.02.3/lib/bytes -I /home/opam/.opam/4.02.3/lib/cstruct -I /home/opam/.opam/4.02.3/lib/ocaml -I /home/opam/.opam/4.02.3/lib/ocaml-migrate-parsetree -I /home/opam/.opam/4.02.3/lib/ocaml/compiler-libs -I /home/opam/.opam/4.02.3/lib/ocplib-endian -I /home/opam/.opam/4.02.3/lib/ppx_cstruct -I /home/opam/.opam/4.02.3/lib/ppx_tools_versioned -I /home/opam/.opam/4.02.3/lib/result -I /home/opam/.opam/4.02.3/lib/rresult -I /home/opam/.opam/4.02.3/lib/sexplib -I /home/opam/.opam/4.02.3/lib/xenstore -no-alias-deps -I xen/proto -o xen/proto/conproto.cmi -c -intf xen/proto/conproto.mli)
# File "xen/proto/conproto.mli", line 19, characters 47-65:
# Error: Unbound type constructor result
```

just put it on all of the packages to be safe i think